docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -0,0 +1,373 @@
+| | | name | string | The file name under which the snippet is mounted. The resulting path will be `/files/<name>`. |
+| | | content | string | The actual text snippet to mount as file.|
+|**status** | | | | **The "status" parameter (not modifiable) holds relevant information about the status of the pipeline. |
+| | conditions | | []object | An array of conditions describing the status of the pipeline.
+| | | lastTransitionTime | []object | An array of conditions describing the status of the pipeline.
+| | | reason | []object | An array of conditions describing the status of the pipeline.
+| | | type | enum | The possible transition types are:<br>- Running: The instance is ready and usable.<br>- Pending: The pipeline is being activated. |
+
+## LogParser specification
+| 1. Parameter | 2. Parameter | 3. Parameter | Type | Description |
+|-|-|-|-|-|
+|**spec** | | | | **The "spec" parameter (modifiable) holds the parser definition.**|
+| | parser | | object | A [Fluent Bit Parsers](https://docs.fluentbit.io/manual/pipeline/parsers). The parser specified here has no effect until it is referenced by a [Pod annotation](https://docs.fluentbit.io/manual/pipeline/filters/kubernetes#kubernetes-annotations) on your workload or by a [Parser Filter](https://docs.fluentbit.io/manual/pipeline/filters/parser) defined in a pipelines filters section. |
+| | | content | string | The actual parser definition in the syntax of Fluent Bit. |
+## Log record processing
+
+After a log record has been read, it is preprocessed by some central configuration sections like the `kubernetes` filter. Thus, when a record is ready to be processed by the sections defined in the LogPipeline definition, it will have several attributes available for processing and shipment.
+
+![Flow](./assets/telemetry-logging-flow.drawio.svg)
+
+Learn more about these attributes in the following sections.
+
+### Container log message
+
+In the example, we assume there's a container `myContainer` of Pod `myPod` running in Namespace `myNamespace` logging to `stdout` with the following log message in JSON format:
+
+```json
+{
+  "level": "warn",
+  "message": "This is the actual message",
+  "tenant": "myTenant",
+  "traceID": "123"
+}
+```
+
+### Tail input
+
+The central pipeline tails the log message from a log file managed by the container runtime. The file name contains the Namespace, Pod, and container information that will be available later as part of the [tag](https://docs.fluentbit.io/manual/concepts/key-concepts#tag). The resulting log record available in an internal Fluent Bit representation looks similar to the following example:
+
+```json
+{
+  "time": "2022-05-23T15:04:52.193317532Z",
+  "stream": "stdout",
+  "_p": "F",
+  "log": "{\"level\": \"warn\",\"message\": \"This is the actual message\",\"tenant\": \"myTenant\",\"traceID\": \"123\"}
+}
+```
+
+The attributes in the example have the following meaning:
+| Attribute | Description |
```

Comment:
This table isn't rendered properly :/

<img width="747" alt="image" src="https://user-images.githubusercontent.com/76950046/179966193-cb8a4f76-55bb-45c9-aec3-0cea5e9d07b1.png">

Reply:
It's less bad now! Although I didn't know THIS is possible 🤣 

<img width="567" alt="image" src="https://user-images.githubusercontent.com/76950046/180188068-222d2827-73e3-412d-94cf-725ce16cce1b.png">

Reply:
fixed